pkg/kn/commands/subscription/create.go

```diff
@@ -26,7 +26,6 @@ import (
 	"knative.dev/client/pkg/kn/commands"
 	"knative.dev/client/pkg/kn/commands/flags"
 	knflags "knative.dev/client/pkg/kn/flags"
-	knmessagingv1 "knative.dev/client/pkg/messaging/v1"
```

Comment:
removing this line seems to break the tests.

Reply:
Should i remove this line

Reply:
No, it's used here: https://github.com/knative/client/blob/a7418fdb92d0b76b356c1cf43f020d46182d589c/pkg/kn/commands/subscription/create.go#L73

Reply:
Only the example message needs to be changed in the source. Imports should be untouched, not needed to be changed at all.